lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -369,16 +413,21 @@ export class UserAgentApplication {
    */
   // TODO: params to accept AuthRequest object instead
   acquireTokenRedirect(request: AuthenticationParameters): void {
```

Comment:
@sameerag looks like the comments for these methods didn't get updated. Please track this in a separate PR.

Reply:
@manoj-rath they are updated in the final integrated PR.

Reply:
@manoj-rath they are updated in the final integrated PR.